src/søknad/barna-dine/BarnaDine.less

```diff
@@ -0,0 +1,129 @@
+.barna-dine {
+    width: 620px;
```

Comment:
Antar at dette er likt for hele søknaden. Bør det settes rundt alle komponentene istedet? Mener å huske at vi gjorde det for kontantstøttesøknaden.

Reply:
Størrelsen den hvite firkanten rundt alle stegene er alltid det samme, men for at barnekortene (som har en gitt størrelse) skal være midtstilt, tror jeg også vi må sette en egen bredde på disse:

![Skjermbilde 2020-02-06 kl  13 45 54](https://user-images.githubusercontent.com/1413265/73938172-0367c900-48e7-11ea-96d9-825f84e6c0cf.png)